### DISCUSSION TITLE
Command Palette Navigation

### DISCUSSION THREAD

**Discussion by att14** [community/community#15589] [Other Feature Feedback, Questions, & Ideas]
Please allow for `ctrl+p`/`ctrl+n` (on macOS), analogous to up and down arrows, for navigation within command palette.

It works on other navigation dropdowns, such as "Search or jump to...", already.

I searched for an existing discussion, but couldn't find it, so I apologize in advance if this is a duplicate.

**Comment by BananaAcid** [comment]
Out of curiosity isn't it `print` and `new window` key bindings you are referring to? Why would you choose these to navigate?

**Reply by att14** [reply]
Generally, on UNIX machines, `ctrl+p` and `ctrl+n` are used for up arrow and down arrow. Especially on macOS where `ctrl` isn't used for any of the commands like print and new window.

**Reply by BananaAcid** [reply]
Ah thx, now knowing where to look, I found [What is the historical origin of CTRL + P for up and CTRL + N for down?](https://unix.stackexchange.com/questions/276759/what-is-the-historical-origin-of-ctrl-p-for-up-and-ctrl-n-for-down) if anyone else is interested :)
